src/core/stream_recv.c

```diff
@@ -1049,18 +1044,17 @@ QuicStreamReceiveComplete(
     _In_ uint64_t BufferLength
     )
 {
-    if (!Stream->Flags.ReceiveCallPending) {
+    if (!Stream->RecvCallPendingCount) {
         return FALSE;
     }
+    Stream->RecvCallPendingCount--;
```

Comment:
This isn't going to work. With multiple receives, we want to support completing multiple with a single call too.